src/OceanSeaIceModels/InterfaceComputations/assemble_net_fluxes.jl

```diff
@@ -56,13 +60,13 @@ function compute_net_ocean_fluxes!(coupled_model)
     freshwater_flux = atmosphere_fields.Mp.data
 
     ice_concentration = sea_ice_concentration(sea_ice)
-    ocean_salinity = ocean.model.tracers.S
+    ocean_salinity = get_ocean_state(coupled_model.ocean, coupled_model.interfaces.exchanger).S
```

Comment:
its hard to read, try

```suggestion
    ocean_state = get_ocean_state(coupled_model.ocean, coupled_model.interfaces.exchanger)
    ocean_salinity = ocean_state.S
```